types/walk/index.d.ts

```diff
@@ -0,0 +1,78 @@
+// Type definitions for walk 2.3
+// Project: https://git.coolaj86.com/coolaj86/fs-walk.js
+// Definitions by: Pontus Östlund <https://github.com/poppa>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+import { Stats } from "fs";
+
+export as namespace walk;
```

Comment:
Is this needed?  It looks like the consumers might be expecting a node module.

Reply:
I don't even know what that means? 

I'm not an avid Typescript programmer, but how would I else extend `fs.Stats?`

Reply:
@poppa, I was referring to the export, rather than the import.  `export as namespace` is generally used when the code will be consumed from a webpage or other non-module environment.  If this code will always be consumed as a node module, the exported declarations below will suffice.

Reply:
Ah, but of course. I've removed the `export as namespace walk`.

Reply:
Thanks!